web/src/reactions.js

```diff
@@ -184,18 +190,18 @@ function generate_title(emoji_name, user_ids) {
     context.comma_separated_usernames = usernames.slice(0, -1).join(", ");
     context.last_username = usernames.at(-1);
     if (current_user_reacted) {
-        return $t(
+        return $t_html(
             {
                 defaultMessage:
-                    "You (click to remove), {comma_separated_usernames} and {last_username} reacted with {emoji_name}",
+                    "<div><span>You (click to remove), {comma_separated_usernames}</span> and <span>{last_username}</span> reacted with {emoji_name}</div>",
             },
             context,
         );
     }
-    return $t(
+    return $t_html(
         {
             defaultMessage:
-                "{comma_separated_usernames} and {last_username} reacted with {emoji_name}",
+                "<div><span>{comma_separated_usernames}</span> and <span>{last_username}</span> reacted with {emoji_name}</div>",
```

Comment:
I don't like adding this much HTML into the translation tag blocks.  The `<div>` definitely doesn't need to part of the translation block. I think the best option might be to move these constructions to a handlebars template? I'm not sure.

I'm also not really sure this change as written makes sense -- `comma_separated_usernames` contains multiple usernames anyway, so you're not enclosing each one on its own span regardless.

Reply:
Yes, I also think that it'll make the code more readable. I'll update the PR with the suggested changes.